pkg/skaffold/kubernetes/context/context.go

```diff
@@ -70,15 +65,19 @@ func getCurrentConfig() (clientcmdapi.Config, error) {
 	return cfg, err
 }
 
-// getRawKubeConfig retrieves and caches the raw kubeConfig. The cache ensures that Skaffold always works with the identical kubeconfig,
-// even if it was changed on disk.
-func getRawKubeConfig() (clientcmdapi.Config, error) {
+func getKubeConfig() clientcmd.ClientConfig {
```

Comment:
If this will be kept, this should have a clarifying comment similar to `getRawKubeConfig`.